src/main/java/com/internet/shop/web/listener/InjectDependenciesListener.java

```diff
@@ -0,0 +1,31 @@
+package com.internet.shop.web.listener;
+
+import com.internet.shop.lib.Injector;
+import com.internet.shop.security.AuthenticationService;
+import com.internet.shop.service.OrderService;
+import com.internet.shop.service.ProductService;
+import com.internet.shop.service.ShoppingCartService;
+import com.internet.shop.service.UserService;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.servlet.annotation.WebListener;
+
+@WebListener
+public class InjectDependenciesListener implements ServletContextListener {
+    @Override
+    public void contextInitialized(ServletContextEvent sce) {
+        Injector injector = Injector.getInstance("com.internet.shop");
```

Comment:
Open-Closed principle violation. 
`Software entities (classes, modules, functions, etc.) should be open for extension, but closed for modification.` 
If we create a new service, we should modify this class and add a new service. Such behavior is a bad practice.

Reply:
got it, thanks) I will delete this